NOTICE.txt

```diff
@@ -1050,3 +1050,120 @@ NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
 LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
 OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+---
+
+## babel
+
+This product contains the following packages of 'babel', a compiler for writing next generation JavaScript: 'babel-core', a Babel compiler core, 'babel-plugin-transform-object-rest-spread', a plugin that allows Babel to transform rest properties for object destructuring assignment and spread properties for object literals, and 'babel-preset-react', a Babel preset for all React plugins.
+'react-addons-css-transition-group', by Babel.
```

Comment:
`react-addons-css-transition-group` is an addon of React. It's already written at React section.